### DISCUSSION TITLE
Unable to abort-build based on --team=name

### DISCUSSION THREAD

**Discussion by jonny-wg2** [concourse/concourse#7422] [General]
We recently moved to concourse 7.4, as well as, adopted teams. With this, we are experiencing issues with aborting a job based on a team name.

For example running this command 

```
> fly -t ci7 abort-build -j deploy/app1 --build 36 --team=deploy
error: unknown flag `team'
```

**Comment by taylorsilva** [comment]
Yup https://github.com/concourse/concourse/issues/5215
Overall, fly could use a good refactor
